knode/knode.go

```diff
@@ -0,0 +1,77 @@
+package knode
+
+import (
+	"errors"
+	"os"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	"k8s.io/client-go/kubernetes"
+	typedcorev1 "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/record"
+
+	"k8s.io/kubernetes/pkg/api/legacyscheme"
```

Comment:
may be all k8s imports can be clubbed together

Reply:
Done.